dapp-developer-guide/getting-started.md

```diff
@@ -0,0 +1,275 @@
+### Meaning of ENS
+
+The Ethereum Name Service (ENS) is a useful tool for dapp developers. ENS is like DNS (Domain Name System), in that it maps a memorable shortcut to an address. Using ENS we can map the friendly name ethereum.eth to the rather unfriendly 0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359. Subsequently, we can use the friendly name in-place of the address, making it easier to remember, and reducing the chance of errors.<br/>
+An ENS name has a root name ethereum.eth which can contain sub-names like wallet.ethereum.eth. Whoever successfully bid for the root name in a Vickrey auction owns it.<br/>
+ENS is the Ethereum Name Service, a distributed, open, and extensible naming system based on the Ethereum blockchain.
+
+### Why Developers need to use the ENS
+ENS doesn't just map to addresses. ENS is highly extensible and supports many types of mappings. It is possible to map the same friendly name to multiple endpoints at the same time. <br/>
+ENS gets rid of the necessity of copying or typing lengthy hexadecimal addresses. With ENS, you’ll be able to send funds to the domain name instead of the long address, engage with your smart contract at mycontract.eth, and check out swarm-hosted sites at swarmsite.eth. <br/>
+
+ENS is a completely decentralized system. Creating new domains under the “.eth” top-level domain is possible by an auction process that takes place on the Ethereum blockchain, and anyone can procure a domain for themselves by taking part.
+
+For example the ethereum.eth name could point to:
+- A multisig wallet contract address
+- Public encryption keys for secure communication
+- Website content (via IPFS hash or multihash)
+
+It could be described with a phone-book metaphor, top-level domains, like ‘.eth’ and ‘.test’ are owned by smart contracts called registrars, which specify rules governing the allocation of their subdomains. Anyone may, by following the rules imposed by these registrar contracts, obtain ownership of a domain for their own use.
+Because of the hierarchical nature of ENS, anyone who owns a domain at any level may configure subdomains - for themselves or others - as desired. For instance, if Alice owns 'alice.eth', she can create 'pay.alice.eth' and configure it as she wishes.
+
+
+#### ENS Subdomains
+They can also give more of an organizational feel to users, where the domain represents the organization and the subdomain resembles the structure of email addresses. A novel use case currently being implemented by Tenzorum Project is to use your subdomain as a username/login to Ethereum ĐApps. <br/>
+Subdomains map names to addresses exactly the same way as domains do, but you don’t need to go through an auction process to create one. Luckily, there is no minimum length limit on ENS subdomains, either. To register a subdomain, you have to be the owner of the domain as a prerequisite.
+
+**Prerequisites**<br/>
```

Comment:
This following section looks to me more like a tutorial
we are considering adding the tutorial section somewhere in the docs
but we don't know yet.

it would be good if you could split this PR in 2 parts, 2 pages

- the first introductory section on what is ENS (that you might want to consider as proposed additions/corrections to the already existent introduction https://docs.ens.domains/, rather than a duplication of the page)

- and a second page dedicated to the tutorial

if you need to put explainers about what ENS is and how it works in certain parts, please also consider quoting and linking to existing material in the documentation